pkg/cloud/wrap/sdkv1.go

```diff
@@ -0,0 +1,68 @@
+package wrap
+
+import (
+	"net/http"
+	"time"
+
+	alierrors "github.com/aliyun/alibaba-cloud-sdk-go/sdk/errors"
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/responses"
+	"github.com/go-logr/logr"
+)
+
+func transformErrorForLog(errIn error) (attrs []any, requestID string, err error) {
+	err = errIn
+	if alierr, ok := err.(alierrors.Error); ok {
+		attrs = append(attrs, "errorCode", alierr.ErrorCode())
+		err = &briefAliError{alierr}
+		if svrerr, ok := alierr.(*alierrors.ServerError); ok {
+			requestID = svrerr.RequestId()
+		}
+	} else {
+		// No error code, just log entire error
+		attrs = append(attrs, "error", err)
```

Comment:
I doubt there will be any other error types, better to add some warning logs here